test/sub_examples/test_examples.py

```diff
@@ -4,14 +4,24 @@
 
 """
 import asyncio
+import pathlib
+import shutil
+import tempfile
+from unittest.mock import MagicMock, patch
 
 import pytest
+import requests
 
 from examples.build_bcd_payload import BcdPayloadBuilder, BcdPayloadDecoder
 from examples.client_async import run_a_few_calls, run_async_client, setup_async_client
 from examples.client_calls import run_async_calls
 from examples.client_custom_msg import run_custom_client
 from examples.client_payload import run_payload_calls
+from examples.contrib.explain import (
```

Comment:
Examples in contrib are not tested with CI. This is due to our policy of not wanting to maintain them.

If you want a maintained example it needs to be directly in examples (and then the review will be tougher, because we need to make sure we can maintain it).

Reply:
Okay gotchu, I will remove the tests entirely.  It was a good exercise, as it pointed out some places that weren't Windows or Python 3.8 compatible

Reply:
Removed all tests, resolving this thread